homeworks/nikolay.krasniuk_Kingstar22/6-typescript/src/index.ts

```diff
@@ -0,0 +1,69 @@
+import './style.css';
+
+const fieldLetterEL:HTMLElement = document.querySelector('[data-characters]');
+const fieldCounterEL:HTMLElement = document.querySelector('[data-caunter-number]');
+const circleEL:HTMLElement = document.querySelector('[data-game-circle]');
+const fieldPlusPoints:HTMLElement = document.querySelector('[data-plus-points]');
+const fieldMinusPoints:HTMLElement = document.querySelector('[data-minus-points]');
+const letters: string = 'QWERTYUIOPASDFGHJKLZXCVBNM'
+let counter:number = 100;
+fieldCounterEL.textContent = `${100}`;
+let valueRandomLetter:string;
+let keyPress:boolean = false;
+let scores: number;
+
+function gameLogic(): void {
+    document.addEventListener('keydown', (event) => {
+        let item = event.key.toUpperCase();
+        if (item === valueRandomLetter) {
+            scores = getRandomNumber(5, 10);
+            counter += scores;
+            changeCircleSize(counter);
+            changeCounterField(counter);
+            fieldPlusPoints.textContent =`+${scores}`;
+        } else if (item !== valueRandomLetter) {
+            scores = getRandomNumber(20, 25);
+            counter -= scores;
+            changeCircleSize(counter);
+            changeCounterField(counter);
+            fieldMinusPoints.textContent =`-${scores}`;
+        }
+        keyPress = true;
+        resultCount();
+    });
+}
+gameLogic();
+
+const readout = setInterval(() => {
+    valueRandomLetter = getRandomLetter(letters)
+    fieldLetterEL.textContent = valueRandomLetter;
+}, 2000);
+
+function getRandomLetter(str:string):string {
```

Comment:
It's better to add spaces before types for readability.

```suggestion
function getRandomLetter(str: string): string {
```